src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb

```diff
@@ -7641,8 +7547,19 @@
     "print(f\"Natural gas flowrate = {m.fs.ngcc.gt.tags_streams['fuel01_F']}\")\n",
     "print(f\"Total load = {m.fs.soec.tags_output['total_electric_power']}\")\n",
     "print(f\"NGCC net power = {m.fs.ngcc.tags_output['net_power']}\")\n",
-    "#print(f\"Thermal input = {0}\")\n",
-    "print(f\"Electric input = {m.fs.soec.tags_output['total_electric_power_per_h2']}\")\n"
+    "print(f\"Electric input = {m.fs.soec.tags_output['total_electric_power_per_h2']}\")\n",
+    "print(f\"GT Power = {m.fs.ngcc.tags_output['gt_power']}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "a38e7084-c83c-41bc-9747-0d5549f6d460",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assert abs(m.fs.soec.tags_output['total_electric_power_per_h2'].value - 37.880) < 1e-1\n",
```

Comment:
These checks are fragile and it appears the test failure is related. I'd suggest using relative checks instead, possibly using `pytest.approx` to check `rel` errors.

Reply:
Yeah, I'll change it.  I did it this way so I didn't have to import pytest, but I guess there is no reason not to.